_team/randy-hart.md

```diff
@@ -0,0 +1,17 @@
+---
+city: Washington
+first_name: Randy
+full_name: Randy Hart
+github:
+last_name: Hart
+name: randy-hart
+role: Procurement specialist
```

Comment:
Procurement Specialist (in keeping with how we've been doing it on other pages).